src/song.ts

```diff
@@ -388,8 +388,6 @@ export class SongManager {
         let song = SongManager.songs.get(id);
         if (typeof song === 'undefined') throw new SongNotFoundError();
 
-        // If the song metadata is old then update it
```

Comment:
If you're going to remove this, might as well make the function synchronous instead